### PR TITLE
Add uploaded file size validation and file upload tests

### PR DESCRIPTION
**Pull request by IvanIvanoff**

### CODE REVIEW

File: lib/sanbase_web/graphql/resolvers/file_resolver.ex [hunk 1]
```diff
@@ -15,47 +17,77 @@ defmodule SanbaseWeb.Graphql.Resolvers.FileResolver do
 
     image_data =
       for %{filename: file_name} = arg <- images, into: [] do
```
Comment (valo) [inline lib/sanbase_web/graphql/resolvers/file_resolver.ex]: Isn't this a simple map with pattern matching? Something line 

```elixir
images
|> Enum.map(fn %{filename: file_name} = arg ->
  arg = %{arg | filename: milliseconds_str() <> "_" <> file_name}
  save_image_content(arg, hash_algorithm)
end)
```

What is the benefit of using for?

File: lib/sanbase_web/graphql/resolvers/file_resolver.ex [hunk 2]
```diff
@@ -15,47 +17,77 @@ defmodule SanbaseWeb.Graphql.Resolvers.FileResolver do
...
+      {:ok, hash}
+    rescue
+      error in File.Error ->
+        %{reason: reason} = error
+        {:error, "Error calculating file's content hash. Reason: #{reason}"}
+    end
   end
 
-  defp save_images_pg(images) do
+  defp save_image_content(%Plug.Upload{filename: file_name} = arg, hash_algorithm)
```
Comment (valo) [inline lib/sanbase_web/graphql/resolvers/file_resolver.ex]: Seems to me that this `hash_algorithm` parameter is not really needed. May be a module attribute will do the trick?

File: lib/sanbase_web/graphql/resolvers/file_resolver.ex (hunk 1 above)
Comment (IvanIvanoff) [reply lib/sanbase_web/graphql/resolvers/file_resolver.ex]: I was using for at some point to do some filtering but it's no longer needed, will change to map

File: lib/sanbase_web/graphql/resolvers/file_resolver.ex (hunk 2 above)
Comment (IvanIvanoff) [reply lib/sanbase_web/graphql/resolvers/file_resolver.ex]: Will be cleaner with module attribute, agreed

**Pull request merged**